src/app/coursegrab/notifications/push_notifications.py

```diff
@@ -165,14 +156,38 @@ def send_emails(section, emails):
     print(f"NOTIFICATIONS : Sending notifications to EMAIL users")
 
     serialized_section = {**section.serialize(), "is_tracking": True}
+    subject_code = serialized_section['subject_code']
```

Comment:
we're sure that this will always exist right! `serialized_section.get("subject_code")` might be better if u dont want scary error but u probs good

Reply:
it doesn't accept null values in db column!